### DISCUSSION TITLE
Local anchors broken in new Github UI

### DISCUSSION THREAD

**Discussion by borkdude** [community/community#62205] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Pages generated with quickdoc (https://github.com/borkdude/quickdoc) contain local anchors (with #). As you can see here: https://github.com/babashka/fs/blob/master/API.md, there is a table of contents above. When you click on one of the links nothing happens. But when you refresh the browser, then it does navigate to the right link. This seems to have broken with the new Github UI. It used to work with the simpler UI that Github had before. Is this a bug in Github or should I change something to quickdoc?

**Comment by axel-schroeder** [comment]
same observation. Anchors do no longer work on github.com
